### ISSUE TITLE
Retail: Odd behavior with adding professions to neuron bars

### ISSUE THREAD

**Issue by teh-fixerer** [brittyazel/Neuron#375]
So far I have found this issue appear when adding "Engineering" and/or "Fishing" from the spell book to a neuron action bar.  Could be a game bug as well.

Steps to reproduce (on a level 50 toon with Kul Tiran Engineering):
Log into game, add Engineering spell from book onto neuron bar.
Click button, nothing happens, no lua errors though.  Tried a "reload' and still does not activate.

I found that if I zone to Stormwind for example the button now works and continues to work for the game session.
If I log off the game and log in again later, the button does not work anymore until I change zones.

Zoning into an instance seems to cause the button to work as well.

For what it is worth if I just type in /cast engineering the same problems are encountered so I am thinking it is a game bug?

Any work arounds or other suggestions are appreciated.  Not a deal-breaker for regular game play - more of an inconvenience thing.

**Comment by jpmeyers** [comment]
I can confirm this issue. As an altaholic I went on a logging spree to find things out.

I am finding that on the first character I log in with I cannot cast the Fishing spell or open some crafting tradeskills (listed below) from a macro, either in chat or via Neuron. If I log into a different character I can then fish or open the tradeskill with Neuron or a chat macro, and continues to work when I return to the initial character. This also works if I zone (the loading screen type), including hearthing.

Tradeskills that seem to be affected this way for me are engineering, black smithing, tailoring, and the fishing spell.

Refreshing, and logging out then in on the same character doesn't affect this initial issue.

With all of that said, and all the alt hopping (full logouts between) I finally decided to turn off all addons and see what would happen. I got exactly the same results. The problem seems to be on Blizz's end, methinks.
